api/libproject/devstart.rb

```diff
@@ -1136,3 +1116,72 @@ def docker_run(cmd_name, args)
   :description => "Runs the specified command in a docker container.",
   :fn => lambda { |*args| docker_run("docker-run", args) }
 })
+
+def create_project_resources(gcc)
+  common = Common.new
+  common.status "Enabling APIs..."
+  for service in SERVICES
+    common.run_inline("gcloud service-management enable #{service} --project #{gcc.project}")
```

Comment:
nit: elsewhere we're using `%W{}` which is an array (with some whitespace rules); use here as well? Here and below

Reply:
I'm beginning to sour on that approach -- if you wind up passing in optional arguments conditionally you wind up having to do this weird array concatenation thing instead of simple string concatenation.

That said we should probably be consistent. Eh. Let me know if you have a preference one way or the other.

Reply:
I don't care very much. @dmohs may have an opinion, but probably don't block on that here. FWIW constructing a command line via array of tokens doesn't seem too unusual to me.

Reply:
If you use string concatenation, you have to worry about spaces in arguments. I don't like worrying about such things. If I have, `my_arg = "hello world"`, this will fail: `run_inline "some_command #{my_arg}"` but this will work reliably: `run_inline %W{some_command #{my_arg}}` as would, `run_inline %W{some_command} + [my_arg]`

Reply:
Done.